code/_onclick/hud/plane_master.dm

```diff
@@ -110,8 +156,43 @@
 /atom/movable/screen/plane_master/emissive_blocker
 	name = "emissive blocker plane master"
 	plane = EMISSIVE_BLOCKER_PLANE
+	layer = EMISSIVE_LAYER
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 	render_target = EMISSIVE_BLOCKER_RENDER_TARGET
+	/// Basically a second copy of the `render_target` because `initial(plane_master_typepath.render_target)` always returns `null` because _BYOND_
+	var/blocker_target = EMISSIVE_BLOCKER_RENDER_TARGET
+
+/// The emissive blocker plane master used by structures to block... something. Presumably whatever's under turfs. (NOTE: Not actually implemented because adding this to every turf would be expensive and there's no reason to use this yet)
+/atom/movable/screen/plane_master/emissive_blocker/turf
+	name = "turf emissive blocker plane master"
+	plane = TURF_EMISSIVE_BLOCKER_PLANE
+	layer = EMISSIVE_TURF_LAYER
+	render_target = TURF_EMISSIVE_BLOCKER_RENDER_TARGET
+	blocker_target = TURF_EMISSIVE_BLOCKER_RENDER_TARGET
+
+/// The emissive blocker plane master used by structures to block emissive turfs and turf overlays
+/atom/movable/screen/plane_master/emissive_blocker/structure
+	name = "structure emissive blocker plane master"
+	plane = STRUCTURE_EMISSIVE_BLOCKER_PLANE
+	layer = EMISSIVE_STRUCTURE_LAYER
+	render_target = STRUCTURE_EMISSIVE_BLOCKER_RENDER_TARGET
+	blocker_target = STRUCTURE_EMISSIVE_BLOCKER_RENDER_TARGET
+
+/// The emissive blocker plane master used by items to block emissive turfs, structures and overlays thereof
+/atom/movable/screen/plane_master/emissive_blocker/item
+	name = "item emissive blocker plane master"
+	plane = ITEM_EMISSIVE_BLOCKER_PLANE
+	layer = EMISSIVE_ITEM_LAYER
+	render_target = ITEM_EMISSIVE_BLOCKER_RENDER_TARGET
+	blocker_target = ITEM_EMISSIVE_BLOCKER_RENDER_TARGET
+
+/// The emissive blocker plane master used by items to block emissive turfs, structures, items and overlays thereof
+/atom/movable/screen/plane_master/emissive_blocker/mob
```

Comment:
```suggestion
/atom/movable/screen/plane_master/emissive_blocker/mob_emissive
```
same for `/mob`, and to be fair I'd do the same for `item` and `structure` as well.

Reply:
Done